pkgs/servers/matrix-synapse/matrix-synapse.patch

```diff
@@ -1,20 +0,0 @@
-diff --git a/homeserver b/homeserver
-new file mode 120000
-index 0000000..2f1d413
---- /dev/null
-+++ b/homeserver
-@@ -0,0 +1,1 @@
-+synapse/app/homeserver.py
-\ No newline at end of file
-diff --git a/setup.py b/setup.py
-index b00c2af..c7f6e0a 100755
---- a/setup.py
-+++ b/setup.py
-@@ -92,6 +92,6 @@ setup(
-     include_package_data=True,
-     zip_safe=False,
-     long_description=long_description,
--    scripts=["synctl"] + glob.glob("scripts/*"),
-+    scripts=["synctl", "homeserver"] + glob.glob("scripts/*"),
```

Comment:
@nyanloutre What was the reason for removing this?

Reply:
This is because there isn't any `homeserver` script anymore

Reply:
The patch was actually creating this wrapper script.
I've opened a PR to restore it.

Reply:
But still, it's better if we can avoid a patch

Reply:
Is there a reason why this patch can't be upstreamed? I'd much prefer the patch to my (more complicated) solution.

Reply:
https://github.com/matrix-org/synapse/issues/1075